src/main/java/com/swrve/segment/SwrveIntegration.java

```diff
@@ -94,7 +94,9 @@ private Map<String,String> flatten(Map<String,Object> properties) {
     Map<String, String> payload = new HashMap<>();
     for (String key: properties.keySet()) {
       Object value = properties.get(key);
-      if(value != null){ // drop null-valued properties
+      if(value == null) { // drop null-valued properties
+        SwrveLogger.e("Dropping null value from TrackPayload properties.");
```

Comment:
~ Prob no harm in actually giving the key name in the log.

Reply:
ok - added that